examples/updatecli.generic/jenkins/weekly.tpl

```diff
@@ -23,6 +23,7 @@ conditions:
   imageTag:
     name: "jenkins/jenkins docker image set"
     kind: yaml
+    disableSourceInput: true
```

Comment:
Nitpick: do you mind renaming this file to `weekly.yaml` ?

Reply:
I renamed the exmaple extension from .tpl to .yaml but additional work is need to cleanup those files